web/app.go

```diff
@@ -186,6 +186,7 @@ func NewAppWithDeps(config *Config, deps Dependencies) (*App, error) {
 		apiGroup.POST("/clusters/:id/tags", ApiClusterCreateTagHandler(deps.clustersService, deps.tagsService))
 		apiGroup.DELETE("/clusters/:id/tags/:tag", ApiClusterDeleteTagHandler(deps.clustersService, deps.tagsService))
 		apiGroup.GET("/clusters/:cluster_id/results", ApiClusterCheckResultsHandler(deps.consul, deps.checksService))
+		apiGroup.GET("/clusters/settings", ApiGetClustersSettingsHandler(deps.clustersService))
```

Comment:
I'm super confused with this.
We already have the `"/checks/:id/settings"` end point for this. Where the `id` is the ansible group id (as we need to group the ansible checks by group0.
This was named on purpose this way, as we were already seeing high changes to have other checks groups (like SAP system wide checks, or SAP landscapes wide checks)